Fixing spelling, grammar, and one markup error, in changes.t2t

XLTechie opened the pull request:
Now that #12376  has been merged, I thought it was time for this.

### Link to issue number:

### Summary of the issue:

While reading through changes.t2t looking for something, I began finding lots of spelling, grammar, and other little errors that could be fixed. I decided to run the whole thing through a spelling and capitalization checker, and touch-up various bits I found wanting along the way.

### Description of how this pull request fixes the issue:

* Fixed a T2T list error.
* Updated URLs from http to https for links that now work with https.
* Numerous spelling, capitalization, and pluralization fixes.
    - Various conversions to American/International English.
    - Combinations using the NVDA key were commonly written uncapitalized; capitalized the "NVDA" portion of such combinations.
    - Corrected the spelling of "Pinnacle TV software" based on probable results in a web search for "Pinacle TV software".
* Made several grammar corrections and clarifications while looking for spelling issues, etc. By no means complete.
    - Corrections from "addon" to "add-on".
    - Removed some unnecessary words from "Attempt to cancel speech for expired focus events" sub-entry.
    - Removed a double negative.

### Testing strategy:

* Strategically read the text around every spelling correction, making certain it wasn't part of a variable, function, etc. name, or other thing that was supposed to look that way.
* Looked up various company names to check spellings and capitalizations, when I didn't already know.

### Known issues with pull request:

There are two styles for second level lists.
43 entries looking like this:
```
- Outer list item.
  - Inner list item.
```
Which is, as I understand it, the T2T spec way of writing inner lists (two space indent for each level).
Then there is the other way:
```
- Outer list item.
 - Inner list item.
```
(One space indent) of which there are 230 examples.
Presumably this later form works fine, even though it is contra-standard.
I did not change either version of the sublist to the other.

### Change log entries:

Changed the change log? I don't think so. No entry needed.

### Code Review Checklist:
-X[ ] Pull Request description is up to date.
- [ ] Unit tests.
- [ ] System (end to end) tests.
- [ ] Manual tests.
-X[ ] User Documentation.
- [ ] Change log entry.
- [ ] Context sensitive help for GUI changes.

XLTechie committed:
Numerous spelling, capitalization, and pluralization fixes.
- Various conversions to American/International English.
- Combinations using the NVDA key were commonly written uncapitalized; capitalized the "NVDA" portion of such combinations.
- Corrected the spelling of "Pinnacle TV software" based on probable results in a web search for "Pinacle TV software".

XLTechie committed:
Made several grammar corrections and clarifications while looking for spelling issues, etc. By no means complete.
- Corrections from "addon" to "add-on".
- Removed some unnecessary words from "Attempt to cancel speech for expired focus events" sub-entry.
- Removed a double negative.

XLTechie committed:
Updated URLs from http to https for links that now work with https.

josephsl commented:
Hi, critical checkup: can translators comment on this PR so the resulting what’s new document is easier to translate? Thanks.

XLTechie commented:
Improving translatability was one of my goals with this, although I could 
probably have spent time to find more items with improvable clarity.
I was also thinking of how long it would take to review, given its length 
as it stands.

I'm not sure it's worth triggering a retranslation of all 290-odd lines of 
changes, as some of them will only be of interest to people who dig far 
back into NVDA's history.

Not being a translator, I can't make an educated guess, only a regular 
one, about what alterations might or might not lead to a clearer document.

I will understand if this is thought to be not worth the effort.

Your opinion?

josephsl commented:
Hi, I think it would be best to focus on 2021.1 changes for now and think about targeting this for 2021.2 or later due to how massive this change is. Also, some items may require digging into old Git commits (some of the oldest commits are based on GNU Bazaar assumptions, so be aware of that). Having worked on what’s new for a number of years (I used to submit PR’s for fixing what’s new document), I understand the temptation to fix old entries – I once tried submitting a PR like this but decided against it due to its size and vast scope. Also, note that changing old what’s new entries will raise issues for some translators who were told they don’t have to translate old entries if they were written prior to the first version of NVDA in which their translation work was included (Korean is a good example in that I intentionally did not translate entries older than 2012.3). Thanks.

zstanecic commented:
Hi Joseph and all!

Me, as a translator, I am against these changes even in the future, because I don’t want to translate old version history of NVDA versions. 

Best,

Zvonimir

 

From: Joseph Lee ***@***.***> 
Sent: Thursday, May 13, 2021 11:53 AM
To: nvaccess/nvda ***@***.***>
Cc: Subscribed ***@***.***>
Subject: Re: [nvaccess/nvda] Fixing spelling, grammar, and one markup error, in changes.t2t (#12412)

 

Hi, I think it would be best to focus on 2021.1 changes for now and think about targeting this for 2021.2 or later due to how massive this change is. Also, some items may require digging into old Git commits (some of the oldest commits are based on GNU Bazaar assumptions, so be aware of that). Having worked on what’s new for a number of years (I used to submit PR’s for fixing what’s new document), I understand the temptation to fix old entries – I once tried submitting a PR like this but decided against it due to its size and vast scope. Also, note that changing old what’s new entries will raise issues for some translators who were told they don’t have to translate old entries if they were written prior to the first version of NVDA in which their translation work was included (Korean is a good example in that I intentionally did not translate entries older than 2012.3). Thanks.




—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub <https://github.com/nvaccess/nvda/pull/12412#issuecomment-840450219> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ACVCDE2RDN6RBAHNQLLIMUDTNOOO7ANCNFSM442GFUQQ> .  <https://github.com/notifications/beacon/ACVCDE2YMBYTEAKZOXOKJ7TTNOOO7A5CNFSM442GFUQ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOGIMEBKY.gif>

XLTechie commented:
I should note the also massive and somewhat related changes proposed by @seanbudd in #12411.

In reading the [Translating Changes](https://github.com/nvaccess/nvda/wiki/TranslatingChanges) wiki page, I misunderstood the "only translate the latest version's changes" section. My assumption was that modifications to the file would only be translated if they occurred in the 2021.1 section. But from the comments above, I gather that all sections since a translation project began would have to be translated retroactively for any changes.

codeofdusk commented:
>Various conversions to American/International English.

Guessing this is something like [Oxford spelling](https://en.wikipedia.org/wiki/Oxford_spelling)?

XLTechie commented:
Bill Dengler wrote:

>       Various conversions to American/International English.
> Guessing this is something like Oxford spelling?

Removed the u from colour, behaviour, etc., for those cases where it 
appeared.
Replaced the S version with the Z version, for words like realise and 
synthesiser.

https://www.bbcamerica.com/anglophenia/2014/07/brits-spell-realize-s
https://www.bbcamerica.com/anglophenia/2014/05/america-drop-u-british-spellings/

codeofdusk commented:
Since NVDA has an international audience, I would do the latter but not the former (i.e. realize, colour, analyse).

XLTechie commented:
Closing this for now, until some reasonable direction for translations is found.
